### PR TITLE
fix(curriculum): update description for HTML fundamentals lecture block

### LINKED ISSUE #56856: [ Full Stack Cert ] - update description for HTML fundamentals lecture block
We have decided on the following update to the intro text here

```md
        "intro": [
           "In these lecture videos, you will learn about HTML fundamentals like working with the <code>id</code> attribute, the <code>class</code> attribute, the <code>div</code> element, the <code>span</code> element, HTML entities, and more."
        ]
```

Here is the file you need to update:

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/i18n/locales/english/intro.json#L1717-L1720

### PR DESCRIPTION
**Pull request by deepANghan**
Checklist:

This PR updates the intro.json file's lecture-html-fundamentals object's property intro
From "Learn about HTML fundamentals in these lecture videos."
To "In these lecture videos, you will learn about HTML fundamentals like working with the <code>id</code> attribute, the <code>class</code> attribute, the <code>div</code> element, the <code>span</code> element, HTML entities, and more."

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56856

### CODE REVIEW

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -1716,7 +1716,7 @@
       },
       "lecture-html-fundamentals": {
         "title": "HTML Fundamentals",
-        "intro": ["Learn about HTML fundamentals in these lecture videos."]
+        "intro": ["In these lecture videos, you will learn about HTML fundamentals like working with the <code>id</code> attribute, the <code>class</code> attribute, the <code>div</code> element, the <code>span</code> element, HTML entities, and more."]
```
Comment (Sembauke) [inline client/i18n/locales/english/intro.json:1719]:
```suggestion
        "intro": ["In these lecture videos, you will learn about HTML fundamentals like the <code>id</code> and <code>class</code> attributes, as well as the <code>div</code> and <code>span</code> elements, HTML entities, and more."]
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Comment by Sembauke** [comment]
@deepANghan Please follow our pr title convention https://contribute.freecodecamp.org/how-to-open-a-pull-request/#prepare-a-good-pr-title - other PRs will give you decent examples as well!

File: client/i18n/locales/english/intro.json (hunk 1 above)
Comment (deepANghan) [reply client/i18n/locales/english/intro.json]: okay. done it

**Comment by deepANghan** [comment]
I have made appropriate changes
Is it okay now?

**Review by Sembauke — APPROVED** [review]

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -1716,7 +1716,7 @@
       },
       "lecture-html-fundamentals": {
         "title": "HTML Fundamentals",
-        "intro": ["Learn about HTML fundamentals in these lecture videos."]
+        "intro": ["In these lecture videos, you will learn about HTML fundamentals like the <code>id</code> and <code>class</code> attributes, as well as the <code>div</code> and <code>span</code> elements, HTML entities, and more."]
```
Comment (gikf) [inline client/i18n/locales/english/intro.json:1719]:
```suggestion
        "intro": [
          "In these lecture videos, you will learn about HTML fundamentals like the <code>id</code> and <code>class</code> attributes, as well as the <code>div</code> and <code>span</code> elements, HTML entities, and more."
        ]
```
To make prettier happy.

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @deepANghan. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**